src/sentry/templates/sentry/emails/activity/generic.html

```diff
@@ -28,7 +28,7 @@ <h2>{{ activity_name }}</h2>
 
   {% endblock %}
 
-  {% if group %}
+  {% if group and not enhanced_privacy %}
```

Comment:
Why do we hide the group information here?

Reply:
The enhanced privacy setting says it hides PII and source code. Does the stacktrace in the group information count as source code?

Reply:
Yep, definitely would. Sounds good to hide it here then!